cfme/tests/test_rest_manual.py

```diff
@@ -212,3 +217,81 @@ def test_update_roles_via_rest_name_change():
             2. Server name remains the same.
     """
     pass
+
+
+@pytest.mark.customer_scenario
+@pytest.mark.meta(coverage=[1661445])
+@pytest.mark.tier(1)
+@test_requirements.rest
+def test_authorization_header_sql_response():
+    """
+    Bugzilla:
+        1661445
+        1686021
+
+    Polarion:
+        assignee: pvala
+        casecomponent: Rest
+        initialEstimate: 1/4h
+        testSteps:
+            1. GET /api/auth?requester_type=ui HEADERS: {"Authorization": "Basic testing"}
+                Perform this GET request with requests.get()
+        expectedResults:
+            1. There should be no sql statement in the response.
+                Expected Response:
+                {
+                "error": {
+                    "kind": "unauthorized",
+                    "message": ("PG::CharacterNotInRepertoire: ERROR:"
+                                "  invalid byte sequence for encoding \"UTF8\": 0xb5\n:"),
+                    "klass": "Api::AuthenticationError"
+                    }
+                }
+    """
+    # Check if it is possible to perform this testing with manageiq-api-client instead of requests.
+    pass
+
+
+@pytest.mark.customer_scenario
+@pytest.mark.meta(coverage=[1682739])
+@pytest.mark.tier(1)
+@test_requirements.rest
+def test_vm_compliance_attribute_rest():
```

Comment:
Hey same question about ```@test_requirements.rest```. Please define it globally

Reply:
Same as above.